### PR TITLE
test: e2e gno organization deployer

### PR DESCRIPTION
**Pull request by MikaelVallenet**
E2E Test Organization Creation.

This PR includes some changes:
- Update generated code for deploy organization
- Update the deploy pkg function to make it functional
- Handle add pkg message type in Adena Mock for test
- Write E2E test for organization deployment

### CODE REVIEW

File: cypress/e2e/dao-creation.cy.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    cy.contains("Next: Configure voting").click();
+    cy.contains("Next: Set tokens or members").click();
+
+    cy.get(
+      '.r-display-6koalj > :nth-child(1) > .r-WebkitOverflowScrolling-150rngu > .r-padding-1jgu3lx > .r-marginBottom-1ifxtd0 > .r-flex-6wfxan > :nth-child(1) > [style="flex-direction: row; width: 100%;"] > :nth-child(1) > [style="width: 100%;"] > .r-alignItems-1habvwh > .r-flexDirection-18u37iz > [style="flex: 1 1 0%; margin-right: 12px;"] > .css-textinput-11aywtz',
+    ).type(memberAddress);
+
+    cy.get(
+      '.r-display-6koalj > :nth-child(1) > .r-alignItems-obd0qt > .r-cursor-1loqt21 > [style="height: 48px; opacity: 1;"] > [style="width: 100%; height: 100%; border-width: 1px; border-color: rgba(0, 0, 0, 0); padding: 0px 20px; align-items: center; justify-content: center;"]',
+    ).click();
```
Comment (n0izn0iz) [inline cypress/e2e/dao-creation.cy.ts:37]: don't use those hyper-specific selectors please, add a testId if needed instead

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

File: packages/utils/gno.ts [hunk 2]
```diff
@@ -35,6 +35,7 @@ export const adenaDoContract = async (
     gasWanted: opts?.gasWanted === undefined ? 10000000 : opts.gasWanted,
     memo: opts?.memo,
   };
+  console.log(JSON.stringify(req));
```
Comment (Villaquiranm) [inline packages/utils/gno.ts:38]: To remove ?

File: packages/context/WalletsProvider/gnotest/index.tsx [hunk 3]
```diff
@@ -268,7 +270,7 @@ const setupAdenaMock = () => {
           throw new Error("Wallet not connected");
         }
         const msg = req.messages[0];
-        if (msg.type !== "/vm.m_call") {
+        if (msg.type !== "/vm.m_call" && msg.type !== "/vm.m_addpkg") {
           throw new Error("Unsupported message type: " + msg.type);
```
Comment (Villaquiranm) [inline packages/context/WalletsProvider/gnotest/index.tsx:274]: Maybe create a constant for `/vm.m_call` and  `/vm.m_addpkg`
on utils/gno.ts ?

**Comment by n0izn0iz** [comment]
this could be a good time to move dao related fields from `GnoNetworkInfo` type to a new `NetworkFeature` `GnoDAOs` type but we can do it in another PR

File: packages/utils/gnodao/deploy.ts [hunk 4]
```diff
@@ -74,35 +71,33 @@ const generateDAORealmSource = (networkId: string, conf: GnoDAOConfig) => {
...
+			// TODO: add a router to support multiple proposal modules
+			propMod := core.ProposalModules()[0]
+			return proposal_single.NewUpdateSettingsHandler(propMod.Module.(*proposal_single.DAOProposalSingle))
+		},
+		func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
+			return tori.NewMintToriHandler()
+		},
+		func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
+			return tori.NewBurnToriHandler()
+		},
```
Comment (n0izn0iz) [inline packages/utils/gnodao/deploy.ts:88]: normal daos don't need to (and can't) control the tori coin, it's only in the demo dao

File: packages/utils/gnodao/deploy.ts [hunk 5]
```diff
@@ -27,33 +27,30 @@ const generateDAORealmSource = (networkId: string, conf: GnoDAOConfig) => {
     dao_core "${network.daoCorePkgPath}"
     dao_interfaces "${network.daoInterfacesPkgPath}"
     proposal_single "${network.daoProposalSinglePkgPath}"
+    "${network.daoUtilsPkgPath}"
     voting_group "${network.votingGroupPkgPath}"
-    "${network.groupsPkgPath}"
-    modboards "${network.modboardsPkgPath}"
     "${network.daoRegistryPkgPath}"
+    "${network.toriPkgPath}"
```
Comment (n0izn0iz) [inline packages/utils/gnodao/deploy.ts:33]: leftover

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

**Review by n0izn0iz — APPROVED** [review]
good for a first step towards stabilizing dao feature

**Pull request merged**